apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt

```diff
@@ -0,0 +1,37 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.get
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class ListAssertionSamples {
+
+    @Test
+    fun get() {
```

Comment:
Please use again the same naming schema, getFeature here and `get` where one can pass an assertionCreator-lambda

Reply:
ok